projects/arlas-components/src/lib/tools/coordinates.tools.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed to Gisaïa under one or more contributor
+ * license agreements. See the NOTICE.txt file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Gisaïa licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { Observable } from 'rxjs';
+
+export class PointFormGroup extends FormGroup {
+
+    public latitude: FormControl;
+    public longitude: FormControl;
+
+    public latitudeChanges$: Observable<any>;
+    public longitudesChanges$: Observable<any>;
+
+    public constructor(initLat: number | string, initLng: number | string) {
+        // eslint-disable-next-line max-len
+        const coordinatesRegex = '^(?<decimal>[+-]?([0-9]*[.])?[0-9]+)$|^(?<degrees>(-?)[0-9]+)°[ ]*((?<minutes>[0-9]+)\'[ ]*((?<seconds>[0-9]+)\")?)?$';
```

Comment:
Could be exported as a const to be defined only once

Reply:
fixed